blueflood-http/src/main/java/com/rackspacecloud/blueflood/tracker/Tracker.java

```diff
@@ -123,6 +127,16 @@ public static void track(HttpRequest request) {
         }
     }
 
+    static String findTid( String uri ) {
+
+        Matcher m = patternGetTid.matcher( uri );
+
+        if( m.matches() )
+            return m.group( 1 );
+        else
+            return null;
```

Comment:
It is more of a best practice. Dont know how relevant it is to this scenario. We can keep the old behavior as it is.